### ISSUE TITLE
User Registration

### ISSUE THREAD

**Issue by sgaseretto** [datashaman/wifidog-auth-flask#1]
Does this authentication server implement user registration and mail confirmation?

**Comment by datashaman** [comment]
No, the user submitted info is purely for identification in the store or coffee shop. No user registration is required, just the voucher code. I have future plans to add user registration so accounts can be stored per user. A user model exists, it is purely for admin types, though.
